model/multi_doc/base_model.py

```diff
@@ -0,0 +1,29 @@
+from model.base_model import SummModel
```

Comment:
I think it's better to call this file smth like `multi_doc_base_model.py`, so we don't have to use parent dir to determine which is which?